package-lock.json

```diff
@@ -1,3040 +1,8 @@
 {
   "name": "vaccination-search-insights",
   "version": "0.3.1",
-  "lockfileVersion": 2,
```

Comment:
I think this is the reason the diff is so large here, `lockfileVersion = 1` uses `dependencies` while v2 uses `packages`.  You may want to check your npm version.  If I run `npm update path-parse` I do get a diff with just that package updated.

Also, we might as well just do a full `npm update`.  This updates a lot of packages, but it didn't appear to break anything.